kibana-reports/public/components/report_definitions/edit/edit_report_definition.tsx

```diff
@@ -31,19 +31,30 @@ import { ReportTrigger } from '../report_trigger';
 export function EditReportDefinition(props) {
   const reportDefinitionId = props['match']['params']['reportDefinitionId'];
   let editReportDefinitionRequest = {
-    report_name: '',
-    report_source: '',
-    report_type: '',
-    description: '',
     report_params: {
-      url: '',
-      report_format: '',
+      report_name: '',
+      report_source: '',
+      description: '',
+      core_params: {
+        base_url: '',
+        report_format: '',
+        time_duration: '',
+      },
     },
-    delivery: {},
-    trigger: {},
+    //TODO: add this field back when the notification module became available
+    // delivery: {},
+    trigger: {
+      trigger_type: '',
+    },
+  };
+
+  let timeRange = {
+    timeFrom: new Date(),
+    timeTo: new Date(),
   };
 
   const editReportDefinition = async (metadata) => {
+    delete metadata.delivery;
```

Comment:
this is temporary, right? It's better to leave a comment with TODO

Reply:
Yep, this is temporary- will leave a TODO comment, good catch